pytorch_lightning/plugins/training_type/rpc_sequential.py

```diff
@@ -56,6 +56,10 @@ def __init__(
 
         .. _RPCSequentialPlugin: https://arxiv.org/abs/1811.06965
 
+        .. deprecated::
+               This plugin has been deprecated. Please use the ``FullyShardedPlugin`` which provides better performance
+               and scaling without pipelining the model.
+
```

Comment:
Not sure what's the correct pattern here. I don't think we'll remove the rpc sequential plugin but it will be unsupported eventually. cc @carmocca

Reply:
We're going to nuke this in a separate PR